app/services/monitor_index_jobs.rb

```diff
@@ -38,18 +38,20 @@ def notify_sentry_for_dead_letter_queue
 
         @processed_from_dead += 1
       rescue => ex
-        Raven.capture_exception(ex)
-        log_error("dead job error #{ex.message} process #{dead_job.inspect}")
+        Raven.capture_exception(ex, :extra => dead_job.inspect)
+        Rails.logger.error("dead job error #{ex.message} process #{dead_job.inspect}")
       end
     end
   end
 
   def reset_desired_capacity_if_needed
+    return unless Rails.env.production?
```

Comment:
Why is this here?  This service seems to do a lot of other things that are AWS-only, so unclear why this method is different.